malaria-ios/Managers/Achievements/Specialized Achievement Managers/Medicine /MedicineAchievementManager.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+
+/// Class that handles all the achievements related to the Medicine object.
+
+class MedicineAchievementManager: NSObject, SpecializedAchievementManager {
+  
+  private let StayingSafeDescription = "Take your first medicine."
+  
+  private var medicine: Medicine
+  private let achievementManager = AchievementManager.sharedInstance
+  private var context: NSManagedObjectContext!
+
+  let tag = "Pills"
+
+  init(medicine: Medicine) {
+    self.medicine = medicine
+    self.context = medicine.managedObjectContext
+    
+    super.init()
+    
+    achievementManager.addTag(tag)
```

Comment:
ditto. Reason: the tags have to be defined anyway therefore it might as well spare this line of code throughout the app by defining directly in the achivement.define method